tests/integration/tokenomics/token_logic_modules/permute_test.go

```diff
@@ -0,0 +1,98 @@
+package token_logic_modules
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestPermute checks the correctness of the permute function such that it is
```

Comment:
#PUC what permute permutates

Reply:
The `permute` has its own godoc which explains it.